csharpProject_efcore/app/console-app/Program.cs

```diff
@@ -0,0 +1,6 @@
+﻿using console_app;
+using library;
+using library.controllers;
+
+new Controller(Resource1.StudentsPath, Resource1.ScoresPath, new ConsoleOutput(), 
+    new DatabaseInitializer(), new DatabaseQueryForFindingANumberOfBestAverages(3)).Run();
```

Comment:
https://docs.microsoft.com/en-us/visualstudio/ide/reference/wrap-indent-align-refactorings?view=vs-2022#wrap-indent-and-align-parameters-or-arguments

indent wrapped parameters